validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/ValidatorLoader.java

```diff
@@ -46,13 +47,25 @@
     validators.putAll(createLocalSignerValidator(config));
     validators.putAll(createExternalSignerValidator(config));
 
-    LOG.debug(
-        "Loaded validators: {}",
-        () ->
-            validators.values().stream()
-                .map(Validator::getPublicKey)
-                .map(BLSPublicKey::toString)
-                .collect(joining(", ")));
+    if (validators.size() > 64) {
```

Comment:
did you decide against the config flag on whether to print ?

Reply:
i think it's probably overkill to start with . I haven't confirmed this with @ajsutton but i think hard coding a value will suffice for now.

Reply:
Yeah I'd just hard code it.

Reply:
64 is slightly weird number to pick mind you.  Maybe if it's 100 or above?